src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/transformers/ParkingTransformer.kt

```diff
@@ -11,13 +11,18 @@
 package org.jetbrains.kotlinx.lincheck.transformation.transformers
 
 import org.objectweb.asm.commons.GeneratorAdapter
+import org.objectweb.asm.Opcodes.*
 import org.jetbrains.kotlinx.lincheck.transformation.*
 import sun.nio.ch.lincheck.*
+import java.util.concurrent.locks.LockSupport
 import sun.misc.Unsafe
 
 /**
- * [ParkingTransformer] tracks [Unsafe.park] and [Unsafe.unpark] method calls,
- * injecting invocations of [EventTracker.park] and [EventTracker.unpark] methods.
+ * The ParkingTransformer class is responsible for inserting bytecode instrumentation
```

Comment:
Is it just a refactoring here?

Reply:
If the hiding logic is here, please consider doing it on the trace post-processing phase

Reply:
> Is it just a refactoring here?

I changed the transformer to track both `Unsafe` and `LockSupport` part APIs. 
Besides improving trace, tracking `LockSupport` directly also simplifies another thing in `ManagedStrategy.kt` (see the diff).